Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -528,12 +581,13 @@ async def send_slack_request_async(client: AsyncWebClient, method: str, http_ver
         try:
             demisto.debug(f'Sending slack {method} (async). Body is: {str(body)}')
             if http_verb == 'POST':
-                if file_:
-                    response = await client.api_call(method, files={"file": file_}, data=body)  # type: ignore
+                if file_upload_params:
+                    # Use three-stage file_upload_v2 'wrapper' method in SDK client class in case there are file_upload_params
```

Comment:
```suggestion
                    # When file_upload_params is provided, performs a three-stage `file_upload_v2` wrapper method in Slack's SDK client
```

Reply:
Improved comments and added link to Slack SDK docs in commit 7c5b1ee65136fc655e92a3456301506de163b9e6